templates/search/blocks/search_results_map.html

```diff
@@ -0,0 +1,9 @@
+<div class="ohos-search-results-map-container">
+    <div id="ohos-search-results-map" class="ohos-search-results-map" data-test="{{ default_geo_data }}" data-js-default-lat="{{ default_geo_data.lat }}" data-js-default-lon="{{ default_geo_data.lon }}" data-js-default-zoom="{{ default_geo_data.zoom }}"></div>
+    <div class="ohos-search-results-map__info">
+        <i class="ohos-search-results-map__info-icon fa-solid fa-circle-info"></i>
+        <div>
+            <p>Map view shows only records with location data. <a href="#">View all search results.</a></p>
```

Comment:
`<a href="#">View all search results.</a>`  href is empty over here. Is this in scope?